src/main/java/net/dries007/tfc/types/DefaultPlants.java

```diff
@@ -46,49 +45,121 @@
+            new Plant(TULIP_WHITE, Plant.PlantType.STANDARD, new int[] {0, 1, 1, 2, 2, 2, 2, 3, 4, 4, 4, 5}, false, false, 15f, 25f, -34f, 33f, 100f, 200f, 9, 15, 1, null),
 
-            // Desert Plants
-            new Plant(PETROVSKIA, Plant.PlantType.DESERT, -29f, 32f, 0f, 200f, 12, 15),
-            new Plant(SAGEBRUSH, Plant.PlantType.DESERT, -34f, 50f, 0f, 100f, 12, 15),
+            // Dry Plants
+            new Plant(PEROVSKIA, Plant.PlantType.DRY, new int[] {0, 1, 2, 2, 3, 3, 3, 3, 4, 5, 5, 0}, true, false, 18f, 35f, -29f, 32f, 0f, 200f, 9, 15, 1, null),
+            new Plant(SAGEBRUSH, Plant.PlantType.DRY, new int[] {0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0}, false, false, 18f, 35f, -34f, 50f, 0f, 100f, 12, 15, 1, null),
 
             // Cactus Plants
-            new Plant(CACTUS, Plant.PlantType.CACTUS, -6f, 50f, 0f, 75f, 12, 15),
+            new Plant(BARREL_CACTUS, Plant.PlantType.CACTUS, new int[] {0, 1, 2, 2, 2, 2, 3, 3, 0, 0, 0, 0}, false, false, 18f, 40f, -6f, 50f, 0f, 75f, 12, 15, 3, "blockCactus"),
 
             // Double Plants
-            new Plant(FERN, Plant.PlantType.DOUBLE, -40f, 33f, 300f, 500f, 4, 11),
+            new Plant(OSTRICH_FERN, Plant.PlantType.TALL_PLANT, new int[] {0, 1, 2, 3, 3, 3, 3, 4, 0, 0, 0, 0}, false, false, 10f, 18f, -40f, 33f, 300f, 500f, 4, 11, 2, null),
 
             // Creeping Plants
-            new Plant(BLUE_DAWN, Plant.PlantType.CREEPING, -40f, 25f, 150f, 500f, 12, 15),
-            new Plant(MOSS, Plant.PlantType.CREEPING, -7f, 36f, 250f, 500f, 0, 11),
+            new Plant(MORNING_GLORY, Plant.PlantType.CREEPING, new int[] {0, 0, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2}, false, false, 15f, 30f, -40f, 31f, 150f, 500f, 12, 15, 1, null),
+            new Plant(MOSS, Plant.PlantType.CREEPING, new int[] {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}, false, false, -2f, 15f, -7f, 36f, 250f, 500f, 0, 11, 1, null),
 
             // Floating Water Plants
-            new Plant(LILYPAD, Plant.PlantType.FLOATING, -34f, 38f, 0f, 500f, 4, 15, 1, 1),
+            new Plant(WATER_LILY, Plant.PlantType.FLOATING, new int[] {0, 1, 2, 2, 2, 2, 3, 4, 5, 5, 5, 6}, false, false, 15f, 30f, -34f, 38f, 0f, 500f, 4, 15, 1, 1, 1, null),
 
             // Grass
-            new Plant(RYEGRASS, Plant.PlantType.SHORT_GRASS, -46f, 32f, 150f, 500f, 12, 15),
-            new Plant(SWITCHGRASS, Plant.PlantType.TALL_GRASS, -29f, 32f, 100f, 300f, 9, 15)
+            new Plant(RYEGRASS, Plant.PlantType.SHORT_GRASS, new int[] {0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0}, false, false, 10f, 18f, -46f, 32f, 150f, 300f, 12, 15, 1, null),
+            new Plant(SWITCHGRASS, Plant.PlantType.TALL_GRASS, new int[] {0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0}, false, false, 13f, 25f, -29f, 32f, 100f, 300f, 9, 15, 2, null),
+
+            // Placeholder plants
+            // todo: replace textures, models, and/or add seasonal varieties
```

Comment:
If these aren't implemented I would leave them out until we have textures / models